resources/api/resource.py

```diff
@@ -183,14 +183,12 @@ class ResourceSerializer(ExtraDataMixin, TranslatedModelSerializer, munigeo_api.
 
     def get_can_only_be_reserved_externally(self, obj):
         today = datetime.datetime.today()
-        opening_hours_period_exists = obj.periods.exists()
         resource_has_external_reservation_link = obj.external_reservation_url
-        resource_period_is_not_valid = not obj.periods.filter(end__gte=today).exists()
-        if (
-            not opening_hours_period_exists or
-            resource_has_external_reservation_link or
-            resource_period_is_not_valid
-        ):
+        resource_period_is_valid = obj.periods.exists() and obj.periods.filter(end__gte=today).exists()
```

Comment:
Shouldn't this be a `or` instead of `and`

Reply:
I wanted to short circuit it. Wanted to check the second part only if `period` exists.